phone-book/src/main.js

```diff
@@ -0,0 +1,119 @@
+
+
+class PhoneBook {
+  constructor() {
+    this.listHeads = { name: 'Name', cname: 'Last name', email:'Email' };
```

Comment:
probably that mapping isn't necessary and it only makes other code-decisions harder and bound.

Probably it's a static content we shouldn't "virtualize it". But let's imagine it's some kind of "translation" which binds to properties even in that case you bound the order of that listHead.

It's better to make it declarative and create required object tags for table-header and for table-body directly.
Looping over that listHeads makes code pretty imperative

Reply:
I've change my methods.. But I'm still using loops for creating user's row.